host/windows/syscall.c

```diff
@@ -1883,10 +1884,6 @@ int oe_syscall_access_ocall(const char* pathname, int mode)
     {
         free(wpathname);
     }
-    if (pACL)
```

Comment:
Why this removed?

Reply:
pACL is a non-owning pointer into the memory of pSD

Reply:
Got it. Can you squash the commits? Thanks!